script.js

```diff
@@ -1,28 +1,75 @@
   },
-  deal: function(){
-
+  deal: function(){                                             // "deal" key:value function
+    alert("Dealing...");                                        // Text alert
+    game.players.forEach(function(playerName){                  // Grabs from the players array, sets up playerName argument
+      var card = game.deck.pop();                               // Removes the a value from the deck array, and sets as "var card"
+      card.player = playerName;                                 // Sets playerName from player array
+      alert(card.player + " has been dealt the " + card.value + " of " + card.suit + "!");  // Text alert with included values
+      game.hands.push(card);                                    // Pushes card into hands array
+    });
   },
-  findHighestCard: function(){
-
+  findHighestCard: function(){                                  // "findHighestCard" key:value function
+    game.hands.sort(function(cardOne, cardTwo){                 // Grabs from the hands array, sets up cardOne and cardTwo arguments
+      if(cardOne.worth > cardTwo.worth){                        // If cardOne.worth is greater than cardTwo.worth, then...
+        return 1;                                               // Return 1
+      }else{                                                    // Or else
+        return -1;                                              // Return minus 1
+      }
+    });
   },
-  announceWinners: function(){
-
+  announceWinners: function(){                                  // "announceWinners" key:value function
+    var place = game.players.length;                            // The length of the players array is set as var place
+    alert("And the winners are...");                            // Text alert
+    game.hands.forEach(function(card){                          // Grabs from the hands array, sets up card argument
+      alert(card.player + " is number " + place + " with the " + card.value + " of " + card.suit + "!"); // Text alert with included values
+      place = place - 1;                                        // Place equal place minus 1
+    });
   },
-  playANewGame: function(){
-
+  playANewGame: function(){                                     // "playANewGame" key:value function
+    game.buildDeck();                                           // Calls buildDeck(); function
+    game.shuffleDeck();                                         // Calls shuffleDeck(); function
+    game.getMorePlayers();                                      // Calls getMorePlayers(); function
+    game.deal();                                                // Calls deal(); function
+    game.findHighestCard();                                     // Calls findHighestCard(); function
+    game.announceWinners();                                     // Calls announceWinners(); function
   }
 }
+game.playANewGame();                                            // Calls playANewGame(); function
```

Comment:
Great job commentating your code! In general I find it helpful to comment on the *why* rather than the *what*